readme.md

```diff
@@ -456,13 +456,13 @@ Change the `?username=` value to your [Wakatime](https://wakatime.com) username.
 
 ### Demo
 
-[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=willianrod)](https://github.com/anuraghazra/github-readme-stats)
+[![Harlok's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=Harlok&range=all_time)](https://github.com/anuraghazra/github-readme-stats)
```

Comment:
The pull request looks good to me. However, why are you also adding `&range=all_time` here when it is already fixed in:

https://github.com/anuraghazra/github-readme-stats/blob/0057714068719cc0d12ddf6f5069147eecd2ff80/src/fetchers/wakatime-fetcher.js#L17-L19

Reply:
It`s not working at me

Reply:
> The pull request looks good to me. However, why are you also adding `&range=all_time` here when it is already fixed in:
> 
> https://github.com/anuraghazra/github-readme-stats/blob/0057714068719cc0d12ddf6f5069147eecd2ff80/src/fetchers/wakatime-fetcher.js#L17-L19

You're right! It will not necessary after fix.

Reply:
> It`s not working at me

Hi @MullerHub !
Can you explain the problem in more detail?
Maybe with examples links.